### ISSUE TITLE
create transcript codebases in system or user temp directory

### ISSUE THREAD

**Issue by aryairani** [unisonweb/unison#1583]
My thinking is that these (especially with v1 codebase) can be quite large, get slurped up into backups / dropbox when I have not yet ever wanted to keep them.  (I can always reconstruct them by re-running the transcript!)

I find myself jumping through hoops to exclude them from my backups, and not really doing a great job anyway.  If I do want to save one, I can copy it out of /tmp afterwards.

If everyone super wants to save them, we can add a command-line option for where to initialize them sometime in the future.

**Comment by pchiusano** [comment]
Good idea.

**Comment by lsund** [comment]
Hi! I could have a look at this. As far as I understand the task is to change the temp-dir for transcript codebases from `$PWD` to `/tmp` by default?

**Comment by pchiusano** [comment]
Great! I’d check out this package for a platform independent way of creating temp directories: https://hackage.haskell.org/package/temporary

Haven’t used it before myself but it looks promising.

**Comment by lsund** [comment]
Yeah, it turns out that System.IO.Temp was already used. I changed the function call to use the system temp directory in this PR: https://github.com/unisonweb/unison/pull/1607

**Comment by ChrisPenner** [comment]
Closed in https://github.com/unisonweb/unison/issues/1583
